### ISSUE TITLE
Internal ADE Accessibility Short Ticket - Upon loading the Payment History page, focus should be sent to the header instead of the back button.

### ISSUE THREAD

**Issue by mmutia** [department-of-veterans-affairs/va.gov-team#112591]
> Hi all,
> 
> Per internal discussion with the ADE team in regard to sending focus to the header vs the back button upon loading the Payment History page, we recommend that focus is sent to the header instead of the back button. We have decided to make this recommendation based on the following reasons:
> 
> 1. While AT may announce the labeled button to access the Payment History page (along with a brief activation noise), the user may have difficulty understanding that the page itself has loaded without the announcement of the heading. With only the announcement of the labeled button and the activation noise, the user will only be certain that the button has been activated -- not that the page itself has loaded.
> 2. Native gestures are built into AT to return the user back to the previous screen. While custom gestures can be set for this purpose, native gestures are commonly used to exit screens.
> 3. If the concern for locating the back button persists (and while we do know that setting the back button directly under the heading may not be good app design), is it possible that the back button can be located directly after the heading in the focus order? This may solve the potential problem of users being unable to locate the back button while being unaware of the native gestures to exit the current screen.
> 4. General orientation and consistency for the user.
> 
> Please let us know if you have any questions or concerns! 

 _Originally posted by @mmutia in [#10772](https://github.com/department-of-veterans-affairs/va-mobile-app/issues/10772#issuecomment-2992651255)_

**Comment by lakper** [comment]
Ticket opened in relation to feedback that was provided to the mobile app team.

**Comment by lakper** [comment]
Closing ticket as feedback was provided to team
